Update character_t.markdown (#29412)
home-assistant/home-assistant.github.io: source/_faq/character_t.markdown

ha_category: Usage
---

> This error means that you used a tab, rather than two spaces in one of your .yml configuration files. Replace the tab with spaces.

Reviewer: ```suggestion
This error means you used tabs rather than two spaces in one of your YAML configuration files. Replace the tabs with spaces.
```
